corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -555,7 +591,7 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             var self = this;
 
             var urlObject = formplayerUtils.currentUrlToObject();
-            urlObject.setQueryData(self.getAnswers(), false);
+            urlObject.setQueryData(self.getAnswers(), false, false, selectValuesByKeys=self.selectValuesByKeys);
```

Comment:
Feel free to call out of scope for this PR, but named parameters for all the options to `setQueryData` would be great.

Reply:
[ac6cda5](https://github.com/dimagi/commcare-hq/pull/32883/commits/ac6cda562f9e7df2d8444ce79c2e090c2157e84a)